TextAlarm/src/main/java/com/theironyard/controllers/TextAlarmController.java

```diff
@@ -0,0 +1,62 @@
+package com.theironyard.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.stripe.exception.*;
+import com.stripe.model.Charge;
+import com.theironyard.objects.User;
+import com.theironyard.utilities.PasswordStorage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.HttpSession;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Nigel on 8/10/16.
+ */
+@Controller
+@RequestMapping("/")
```

Comment:
No point in this request mapping.  Use it at a class level to make all of the endpoints have it.  At the root it doesn't add anything.  In fact your `/login` turns into `//login` because of this mapping.